linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_login_defs/rule.yml

```diff
@@ -46,14 +46,20 @@ references:
     stigid@sle15: SLES-15-040420
     stigid@ubuntu2004: UBTU-20-010016
 
-ocil_clause: 'the above command returns no output, or if the umask is configured incorrectly'
+ocil_clause: 'the above command returns no output, or the umask is configured incorrectly'
 
 ocil: |-
     Verify the <tt>UMASK</tt> setting is configured correctly in the <tt>/etc/login.defs</tt> file by
     running the following command:
-    <pre># grep -i "UMASK" /etc/login.defs</pre>
+    <pre>$ sudo grep -i "UMASK" /etc/login.defs</pre>
     All output must show the value of <tt>umask</tt> set as shown in the below:
-    <pre># grep -i "UMASK" /etc/login.defs
-    umask {{{ xccdf_value("var_accounts_user_umask") }}}</pre>
+    <pre>umask {{{ xccdf_value("var_accounts_user_umask") }}}</pre>
 
 platform: login_defs
+
+fixtext: |-
+    Configure the {{{ full_name }}} to define default permissions for all authenticated users in such a way that the user can only read and modify their own files.
+
+    Add or edit the lines for the "umask" parameter in the "/etc/login.defs" files to "{{{ xccdf_value("var_accounts_user_umask") }}}":
+
+    umask {{{ xccdf_value("var_accounts_user_umask") }}}
```

Comment:
I'm confused by lower case `umask` here, other part of the file mention `UMASK`, does it matter. The RHEL8 STIG seems to be very inconsistent in regards to the case, some parts are in lower case others in upper case

Reply:
Great, I have found that the man login.defs page uses uppercase and there is nothing about case insensivity or case sensitivity and in the actual /etc/login.defs it's uppercase so I will change everything to uppercase.